[2021-12-21 08:00:02,658] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: rank_update.pull_from_git 2021-12-21T07:50:00+00:00 [queued]>
[2021-12-21 08:00:02,663] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: rank_update.pull_from_git 2021-12-21T07:50:00+00:00 [queued]>
[2021-12-21 08:00:02,664] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 08:00:02,664] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-12-21 08:00:02,664] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 08:00:02,670] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): pull_from_git> on 2021-12-21T07:50:00+00:00
[2021-12-21 08:00:02,671] {standard_task_runner.py:52} INFO - Started process 110242 to run task
[2021-12-21 08:00:02,676] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'rank_update', 'pull_from_git', '2021-12-21T07:50:00+00:00', '--job-id', '1068', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/rank_update.py', '--cfg-path', '/tmp/tmp5tofv_ij', '--error-file', '/tmp/tmp5tm6cdfp']
[2021-12-21 08:00:02,676] {standard_task_runner.py:77} INFO - Job 1068: Subtask pull_from_git
[2021-12-21 08:00:02,712] {logging_mixin.py:104} INFO - Running <TaskInstance: rank_update.pull_from_git 2021-12-21T07:50:00+00:00 [running]> on host airflowvm
[2021-12-21 08:00:02,795] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=rank_update
AIRFLOW_CTX_TASK_ID=pull_from_git
AIRFLOW_CTX_EXECUTION_DATE=2021-12-21T07:50:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-21T07:50:00+00:00
[2021-12-21 08:00:02,796] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-12-21 08:00:02,796] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'git -C /home/airflow/AlgorithmStudy_211124 pull']
[2021-12-21 08:00:02,798] {subprocess.py:75} INFO - Output:
[2021-12-21 08:00:04,372] {subprocess.py:79} INFO - Already up to date.
[2021-12-21 08:00:04,373] {subprocess.py:83} INFO - Command exited with return code 0
[2021-12-21 08:00:04,408] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=rank_update, task_id=pull_from_git, execution_date=20211221T075000, start_date=20211221T080002, end_date=20211221T080004
[2021-12-21 08:00:04,425] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-12-21 08:00:04,433] {local_task_job.py:151} INFO - Task exited with return code 0
